docs/customize/browser-settings.mdx

```diff
@@ -846,7 +846,7 @@ record_video_size: dict | None = None. [Playwright Docs: `record_video_size`](ht
 
 ```
 
-Video size. Example: `{"width": 1280, "height": 720}`
+Video size. Example: `{"width": 1920, "height": 1080}`
```

Comment:
careful, the viewport always needs to be smaller than the window size or content will be cut off, I usually take ~40px off the height and ~5px off the width